index.html

```diff
@@ -1,62 +1,68 @@
-    </head>
-    <header>
-        
-      
-    
-     <nav>
-        <a href="#About/">About</a> |
-        <a href="#Experience/">Experience</a> |
-        <a href="#Connect/">Connect</a> |
-        </nav>
-
-        <h1 id="myName">Manizha khorram</h1>
-
-        
-    </header>
-    <body>
-        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
-        <section>
-            <h2><a href="#About">About</a></h2>
-            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
-        </section>
 
-        <section>
-            <h2><a href="#Experience">Experience</a></h2>
-            <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
-        </section>
+<head>
+    <title>Manizha khorram </title>
+    <meta name="Author" content="Manizha khorram">
+    <link rel="stylesheet" type="text/css" href="Style.css" />
+</head>
+<header>
+    <div id="div4nav">
+    </div>
+    <nav>
+        <a href="#About/">About</a> 
+        <a href="#Experience/" id="experience">Experience</a> 
+        <a href="#Connect/" id="connect">Connect</a> 
+    </nav>
 
-        <section id="skills">
-             <h2><a href="#Connect">Skills</a></h2>
-             <ul></ul>
-             
-        </section>
+    <h1 id="myName">Manizha khorram</h1>
 
-        <section>
-            <h2><a href="#Connect">Connect</a></h2>
-            <ul>
-              <li><a  href="mailto:manizh.khorram@gmail.com">Gmail</a></li>
-              <li><a href="https://github.com/Manizha-khorram">GitHub account</a></li>
-              <li><a href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a></li>
+    <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="myImage">
 
-            </ul>
+</header>
 
-        </section>
+<body>
+
+    <section class="myInfo">
+        <h2><a href="#About">About</a></h2>
+        <p id="about">I love coding. I would like to work as professional sofware developer. I hold bachelor of computer
+            science degree, I love problem solving and creating new things.I keen to learn challenging roles in
+            javaScript, software development and related technologies.</p>
+    </section>
+
+    <section class="myInfo">
+        <h2><a href="#Experience">Experience</a></h2>
+        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University
+            Work Force Development Program) in Afghanistan.</p>
+    </section>
+
+    <section id="skills" class="myInfo">
+        <h2><a href="#Connect">Skills</a></h2>
```

Comment:
I see the section below is "#Connect", I wonder if this should be "#Skills" ? :thinking: :smile: